.github/workflows/main.yml

```diff
@@ -0,0 +1,31 @@
+name: Publish to FTP on Development Push
+
+on:
+  push:
+    branches:
+      - development
+
+jobs:
+  publish:
+    runs-on: windows-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Setup MSBuild
+        uses: microsoft/setup-msbuild@v1.0.2
+
+      - name: Build and publish
+        run: |
+          msbuild /p:Configuration=Release /p:OutputPath=./publish_output
+          echo "::set-output name=publish_path::$(pwd)/publish_output"
+
+      - name: Upload to FTP
+        uses: SamKirkland/FTP-Deploy-Action@3.1.1
+        with:
+          server: ${{ secrets.FTP_SERVER }}
+          username: ${{ secrets.FTP_USERNAME }}
```

Comment:
I added only password as a secret. FTP_SERVER and FTP_USERNAME are under variables.